books/calculus/bake

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env ruby
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :calculus) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  # Update the preface title  TODO: add title method
+  book.pages('$.preface').search("div[data-type='document-title']").each do |title|
+    title.replace_children(with:
+      <<~HTML
+        <span data-type="" itemprop="" class="os-text">#{title.text}</span>
+      HTML
+    )
+    title.name = 'h1'
+  end
+
+  BakeNotes.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeChapterKeyEquations.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeChapterKeyConcepts.v1(chapter: chapter, metadata_source: book.metadata)
+
+    chapter.pages.each do |page|
+      page.search('section.section-exercises').each do |section|
+        section.exercises.each do |exercise|
+          number = exercise.count_in(:chapter)
+          BakeExercises.section_exercises(exercise: exercise, number: number)
+        end
+
+        section.wrap(%(<div class="os-eos os-section-exercises-container" data-uuid-key=".section-exercises">))
+        section_title = I18n.t(:section_exercises, number: "#{chapter.count_in(:book)}.#{section.count_in(:chapter)}")
```

Comment:
Use `page.count_in(:chapter)` -- it will give you the same result but really when you see "1.3" that is talking about page 3 in chapter 1.